tests/preprocessing/test_data.py

```diff
@@ -471,3 +471,45 @@ def test_inverse_transform(self):
         assert_eq_df(df, enc.inverse_transform(enc.transform(df)))
         assert_eq_df(df, enc.inverse_transform(enc.transform(df).values))
         assert_eq_df(df, enc.inverse_transform(enc.transform(df).values))
+
+
+class TestPolynomialFeatures:
+    def test_basic(self):
+        a = dpp.PolynomialFeatures()
+        b = spp.PolynomialFeatures()
+
+        a.fit(X)
+        b.fit(X.compute())
+        assert_estimator_equal(a._transformer, b)
```

Comment:
This should be `assert_estimator_equal(a, b)`. It'll fail until you do the `copy_learned_attributes`.